settings.gradle.kts

```diff
@@ -17,7 +17,7 @@ pluginManagement {
 }
 
 plugins {
-    id("com.gradle.enterprise").version("3.10.3")
+    id("com.gradle.enterprise").version("3.11")
     id("io.github.gradle.gradle-enterprise-conventions-plugin").version("0.7.6")
     id("gradlebuild.base.allprojects")
     id("com.gradle.enterprise.test-distribution").version("2.3.5") // Sync with `build-logic/build-platform/build.gradle.kts`
```

Comment:
Actually, this application needs to go away, and the other build file(s) probably need to be changed now.